peps/pep-0735.rst

```diff
@@ -779,18 +704,47 @@ has impacts on high-level tools like IDEs and Dependabot, which cannot support
 deep integration with these Dependency Groups. (For example, at time of writing
 Dependabot will not flag dependencies which are pinned in ``tox.ini`` files.)
 
+Why not support Dependency Group Includes in ``[project.dependencies]`` or ``[project.optional-dependencies]``?
+---------------------------------------------------------------------------------------------------------------
+
+Earlier drafts of this specification allowed Dependency Group Includes to be
+used in the ``[project]`` table.
+However, there were several issues raised during community feedback which led
+to its removal.
+
+Only a small number of additional use cases would be addressed by the inclusion
+of Dependency Groups, and it increased the scope of the specification
+significantly. In particular, this inclusion would increase the number of parties
+impacted by the addition. Many readers of the ``[project]`` table, including build
+backends, SBOM generators, and dependency analyzers are implicated by a change to
+``[project]`` but may continue to operate as-is in the presence of a new (but
+unconnected) ``[dependency-groups]`` table.
+
+Separately from the above concern, the usage modes for
+``[dependency-groups]`` which rely upon inclusion from the ``[project]`` table
+encourage package maintainers to move package metadata from the currently
+standardized location into a newly standardized location. This has a
+negative effect on the simplicity of the mostly static ``pyproject.toml`` metadata
+definition laid out in :pep:`621`.
```

Comment:
This is a bit confusing as written.

Perhaps... 

> Separately from the above concern, allowing inclusion of dependency groups from the ``[project]`` table encourages package maintainers to move dependency metadata out of the current standard location. This complicates static ``pyproject.toml`` metadata and conflicts with the goal of :pep:`621` to store dependency metadata in a single location.

Reply:
Yes, thank you! I agree that this was poorly phrased -- "newly standardized" is an odd way to put it and your phrasing also sounds more positive about the virtues of the PEP 621 approach. I'm going to take your suggested text verbatim.